guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work@2x.png"
+   alt-text = "An image that illustrates how labels feature in silent payments works."
+   caption = "."
+   width = 1024
+   height = 801
+   layout = "full-width"
+%}
+
+
+These labelling and detection features can also be useful for:
+
+- Invoices: merchant software like BTCPay can use labels to match incoming payments with invoices. This is required because on-chain addresses themselves cannot be predicted on used to make the mapping
+- Static deposit addresses for exchanges: Users looking to deposit bitcoin at exchanges may get confused when they see changing/different deposit addresses. Exchanges can now share static addresses with users; this will keep deposit addresses the same while allowing the exchange to map deposit transactions to users
+
+
+### Contacts
+
+Contacts are another feature that are greatly improved with BIP-352. Since users can safely use the same silent payment address for multiple payments, it is natural for them to want to store these for future use. Contacts are a great way to store them in terms that users can intuit: names and faces/images. The Contacts page here provides good guidance about the topic.
+
+Labels help Contacts in another interesting way by enabling ‘sender contacts’ where the user can create Contacts for parties they only receive bitcoin from. This is accomplished by adding a different label for every contact.
+
+{% include image-gallery.html pages = page.images_contacts %}
+
+Overall, Silent payments present a great opportunity to start basing payments UX in terms of people instead of addresses, and applications.
+
+{% include tip/recommendation.html %}
+
+Wallets should create an in-app contact list and avoid storing bitcoin payment information in the operating system’s default contact list since it cannot guarantee its safety for data breaches or phone hacks.
+
+{% include tip/close.html %}
+
+
+## Setup
+
+Applications can take a variety of approaches to set up and communicate BIP-352 wallets. Since such a wallet has a reusable address, this could be introduced and explained to users during the setup process. Some wallets may use unique or significant locations (mobile widget, custom app logos, watch faces) to house this static address for easy retrieval, and may highlight the same during setup.
+
+Since the BIP-352 comes with some new primitives (term) (spend key, scan key) & processes (like labels, backing up files), the setup process might be a good opportunity to introduce some of these as needed without overloading the user.
+
+{% include image-gallery.html pages = page.images_sp-onboarding %}
+
+One thing to keep in mind: since silent payments impacts UX flows around sending, syncing & scanning, backup, recovery and applications might even choose to allow users to not adopt the silent payments altogether!
```

Comment:
The point about interoperability is good, but it is not clear from this sentence why some wallets would not want to have those features. Is it just too much work? Or are there other downsides?

This could be a more general note about interoperability.

Reply:
Not sure I understood what you meant here, would love to know more...

Reply:
Anyway text has been removed in a commit a few weeks back...